sentry-ruby/lib/sentry/baggage.rb

```diff
@@ -26,11 +26,11 @@ def initialize(items, mutable: true)
     #
     # @param header [String] The incoming Baggage header string.
     # @return [Baggage, nil]
-    def self.from_incoming_header(header)
+    def self.from_baggage_string(baggage_string)
```

Comment:
I'm not sure if this method is supposed to be public or not. If it is public for users, we can't simply rename it as it'll break users' apps.

Reply:
Yeah, you're right. Even if it's not public by design, folks might be using it. I could either make a deprecated wrapper with the old name and keep the new name, or revert the rename, either are fine by me.